README.md

```diff
@@ -133,6 +133,60 @@ Migrating shouldn't be that difficult though. `UUIDTools::UUID` implements
 by default. But it's good to be aware of this in case you're running into
 weirdness.
 
+## Invalid Values
+
+By default, this will raise an exception if an invalid value is attempted to be stored.
+However, the PostgreSQL driver nils invalid UUID values.
+To get PG-friendly behavior, pass in the option `:nil_invalid_values => true` when creating the attribute.
+
+Additionally, as a convenience function, you can add the following to `ApplicationRecord` to get parity between PG and MySQL:
+
+```
+def self.uuid_fields(*fields)
+	if ActiveRecord::Base.connection.adapter_name.downcase.starts_with?("mysql")
+		fields.each do |fld|
+			attribute fld, MySQLBinUUID::Type.new(:nil_invalid_values => true)
+		end
+	end
+end
+```
+
+Then, in your models, just begin them by marking the fields which are UUIDs.
+
+## UUID Type in migrations
```

Comment:
I would like to leave this out as it was intentionally not included with this gem.

It's nice of you though to inform people how they could achieve this. I'd propose moving the example code you wrote into a Gist, and link to it in a new final line of the `## No `uuid` column in database migrations` chapter in the README:

```
If you'd like to have a `uuid` type available in your migrations, take a look at this gist:
---gist-link-goes-here---
```